src/Expectation.php

```diff
@@ -60,6 +60,17 @@ public function and($value): Expectation
         return new self($value);
     }
 
+    /**
+     * Parses Json String to Array.
+     */
+    public function json(): Expectation
+    {
+        Assert::assertIsString($this->value);
```

Comment:
`$this->toBeString();`

Reply:
Seen as `toBeJson()` already does the string check, we don't need this check.